src/main/java/com/crosswordservice/controller/CrosswordController.java

```diff
@@ -0,0 +1,106 @@
+package com.crosswordservice.controller;
+
+import com.crosswordservice.baseClasses.Configuration;
+import com.crosswordservice.baseClasses.Question;
+import com.crosswordservice.crosswordchecker.CrosswordChecker;
+import com.crosswordservice.repositories.ConfigurationRepository;
+import com.crosswordservice.repositories.QuestionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.ObjectInputFilter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Rest Controller for the crossword-puzzle backend
+ */
+@RestController
+public class CrosswordController {
+
+    @Autowired
+    QuestionRepository questionRepository;
+    @Autowired
+    ConfigurationRepository configurationRepository;
+
+    @PostMapping("/inputTestData")
+    public List<Question> inputTestData(){
+        Configuration config = new Configuration("test");
+        configurationRepository.save(config);
+
+        List<Question> questions = new ArrayList<>();
+
+        Question quest1 = new Question(config.getId(),"Which language extends Javascript with type safety?","Typescript");
+        Question quest2 = new Question(config.getId(),"How is the system of rules called which defines well-formed expressions?","Syntax");
+        Question quest3 = new Question(config.getId(),"Which loop allows to set the count of iterations in the head?","for-loop");
+        Question quest4 = new Question(config.getId(),"What is the abbreviation of the computing unit in a computer?","CPU");
+        questions.add(quest1);
+        questions.add(quest2);
+        questions.add(quest3);
+        questions.add(quest4);
+        questionRepository.saveAll(questions);
+
+        return questions;
+    }
+
+    @PostMapping("/create-configuration")
+    public Configuration saveConfiguration(@RequestBody Configuration configuration){
+        configurationRepository.save(configuration);
+        return configuration;
+    }
+
+    @PostMapping("/save-all-questions/{name}")
```

Comment:
just name the path `/questions/{name}` (Never use CRUD function names in URIs)
Also what is `name`? Please add some documentation.